patches/@onfido+react-native-sdk+8.3.0.patch

```diff
@@ -15,3 +15,49 @@ index b4c7106..d5083d3 100644
          }
          classDirectories.setFrom(fileTree(
                  dir: 'build/intermediates/javac/debug/compileDebugJavaWithJavac/classes/com/onfido/reactnative/sdk',
+diff --git a/node_modules/@onfido/react-native-sdk/android/publish.gradle b/node_modules/@onfido/react-native-sdk/android/publish.gradle
```

Comment:
We have a little bit different patch for this, maybe you should do it the same since our PR will also be merged soon most probably 😅 : https://github.com/Expensify/App/pull/31558/files#diff-38078bd96e4f2e196211b4e9eb902a3fd1a02b52238a7476813f92be71492105

Reply:
I'm testing that out!

Reply:
It works with your solution, so I'm going to use it instead 😄